### ISSUE TITLE
Documentation - How to train

### ISSUE THREAD

**Issue by tinohager** [AlexeyAB/darknet#2967]
![image](https://user-images.githubusercontent.com/8215186/56361268-38cfe880-61e7-11e9-9392-796cacdcc82f.png)

What is the point with Pascal VOC Data?

Can we summarize the points under Training

**How to train**
- Pascal VOC Data
- Detect your custom objects
- Detect your custom objects with tiny-yolo
- Multi GPU Support

**Comment by tinohager** [comment]
[How to train (to detect your custom objects):](https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects) -> Move Point `7. Download pre-trained weights for the convolutional layers` to position 1, similar as [How to train (Pascal VOC Data):](https://github.com/AlexeyAB/darknet#how-to-train-pascal-voc-data) download need time and it is the last point

**Comment by tinohager** [comment]
**How to train**

- Test with existing dataset (Pascal VOC data sets)
- Detect your custom objects
  - Changes for tiny-yolo
- Multi GPU Support

**Comment by tinohager** [comment]
Add Info for minimum images:

The general rule: for each object which you want to detect - there must be at least 1 similar object in the Training dataset with about the same: shape, side of object, relative size, angle of rotation, tilt, illumination.
The simplest rule: you should use at least 2000 images per class

https://github.com/AlexeyAB/darknet/issues/2649

**Comment by AlexeyAB** [comment]
Are you about the organization in the form of points and sub-points?

**Comment by AlexeyAB** [comment]
> Add Info for minimum images:
> 
> The general rule: for each object which you want to detect - there must be at least 1 similar object in the Training dataset with about the same: shape, side of object, relative size, angle of rotation, tilt, illumination.
> The simplest rule: you should use at least 2000 images per class

It is here: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> for each object which you want to detect - there must be at least 1 similar object in the Training dataset with about the same: shape, side of object, relative size, angle of rotation, tilt, illumination. So desirable that your training dataset include images with objects at diffrent: scales, rotations, lightings, from different sides, on different backgrounds - you should preferably have 2000 different images for each class or more, and you should train 2000*classes iterations or more

**Comment by tinohager** [comment]
I have started last week to deal with the training of yolo custom objects. I have worked my way through the documentation and I am still missing a straight line. I think structuring the documentation a little more would make sense.

I would suggest to divide the documentation into own areas. My suggestion would be create a doc directory in your repository with the Training.md file and move all the content for training in this new file.

```
/doc/
 -> Training.md
```

Then I would rename the headings after the upper proposition.

Can I send a pull request?
